samples/python/tutorial_code/introduction/display_image/display_image.py

```diff
@@ -0,0 +1,19 @@
+## [imports]
+import cv2 as cv
+import sys
+## [imports]
+## [imread]
+img = cv.imread(cv.samples.findFile("starry_night.jpg"))
+## [imread]
+## [empty]
+if img is None:
+    print("Could not read the image.")
+## [empty]
+## [imshow]
+cv.imshow("Display window", img)
+k = cv.waitKey(0)
+## [imshow]
+## [imsave]
+if k == ord("s"):
+    sys.exit("Could not read the image.")
```

Comment:
Wrong "if" was replaced - this "if" is for saving image.

Reply:
Whops, you're right of course!

Reply:
Whoops you're right of course!